onnx/__init__.py

```diff
@@ -77,7 +77,7 @@ def _serialize(proto: Union[bytes, google.protobuf.message.Message]) -> bytes:
                     "Please use save_as_external_data to save tensors separately from the model file."
                 ) from e
             raise
-        return result
+        return result  # type: ignore
```

Comment:
We should avoid ignores without error nums (too board)

Reply:
mypy returns `Returning Any from function declared to return "bytes"` in this case. I assume SerializeToString() is annotated as Any when this function returns bytes. mypy detects the inconsistency. I don't think we can easliy remove it. I don't know if onnx is the source of the wrong annotation or if it is protobuf.